rotkehlchen/accounting/structures/base.py

```diff
@@ -277,6 +290,18 @@ def get_identifier(self) -> str:
     def get_assets(self) -> list[Asset]:
         return [self.asset]
 
+    def has_details(self) -> bool:
+        if self.extra_data is None:
+            return False
+        return len(self.extra_data.keys() & ALL_DETAILS_KEYS) > 0
+
+    def get_details(self) -> Optional[dict[str, Any]]:
+        if self.extra_data is None:
+            return None
+        details = {k: v for k, v in self.extra_data.items() if k in ALL_DETAILS_KEYS}
+
+        return details if len(details) > 0 else None
```

Comment:
This is not what I asked in the previous comment. Read the diff more carefully. Wrong newline removed :)
```suggestion

        details = {k: v for k, v in self.extra_data.items() if k in ALL_DETAILS_KEYS}
        return details if len(details) > 0 else None
```